src/app/shared/components/batchupload/batchupload.component.html

```diff
@@ -0,0 +1,85 @@
+<ng-template #batchUploadNSS let-c="close" let-d="dismiss">
+    <div>
+        <div class="modal-header">
+            <ng-container><div class="title"> Batch Upload </div></ng-container>
+            <button class="icon-button" (click)="d('closed'); selectUpload = false; clearTable()"><i class="far fa-times"></i></button>
+        </div>
+        <div class="modal-body batch-scrollbars">
+            <div class="upload-type" *ngIf='!selectUpload'>    
+                <button type="button" class="button blue" (click)='selectUpload = true;'>Upload Equations</button>
+            </div>
+            
+            <div *ngIf="!selectUpload" class="upload-type">
+                <h6>For an example template download the file <a href="./assets/example_nss_bulk_upload.xlsx">here</a>.</h6>
+            </div>
+            <!-- Back button -->
+			<button *ngIf='selectUpload' type="button" class="button green" (click)='selectUpload = false; clearTable()'>Back</button>
+			<!-- File selector -->
+			<div class="upload-input">
+                <input *ngIf='selectUpload && !tableDisplay' (change)="selectFile($event)" accept=".xls,.xlsx" type='file' multiple='false'/>
+            </div>
+            <!-- Select worksheet -->
+            <div *ngIf="sheetNamesButtons">
+                <h6 class="title">Select a Worksheet:</h6>
+                <ng-container *ngFor="let sheetName of wb.SheetNames">
+                        <button class="button grey mright-sm mtop-xs" (click)='selectSheet(sheetName)'>{{sheetName}}</button>
+                </ng-container>
+			</div>
+
+            <div *ngIf="tableDisplay">
+                <form class="editScen">
+                    <div class="table-title" *ngFor="let e of equationData, let i = index">
+                        <h4 [ngStyle]="{'color': success[i]}">Equation {{i+1}}</h4>
```

Comment:
I don't think the `[ngStyle]="{'color': success[i]}"` is working, I don't see any colors applied to these headers

Reply:
The color shows up after you click create. I added it to show the user which equations were added and which ones got an error. 
![image](https://user-images.githubusercontent.com/59063181/121917372-a9f18480-ccfa-11eb-9b64-a417a4eae33d.png)